frontend/interactEM/src/hooks/useOperators.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { Operator } from "../operators";
+
+import {
+  readOperators,
+  ReadOperatorsError,
+  ReadOperatorsResponse,
+} from "../client";
+
+const useOperators = () => {
+  const [operators, setOperators] = useState<Operator[] | null>(null);
+  const [error, setError] = useState<ReadOperatorsError | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  useEffect(() => {
```

Comment:
this we should be able to do with a mutation. we can punt, though

Reply:
Sure, let doe this in a follow up.